src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -160,6 +169,96 @@ export class MakeCogGithub {
     return tileSet;
   }
 
+  /**
+   * Given a old and new lds layer stac item and log the changes for pull request body
+   */
+  getVectorChanges(newLayer: StacLink | undefined, existingLayer: StacLink | undefined): string | null {
+    // Update Layer
+    if (newLayer != null && existingLayer != null) {
+      const featureChange = Number(newLayer['lds:feature_count']) - Number(existingLayer['lds:feature_count']);
+
+      if (newLayer['lds:version'] === existingLayer['lds:version'] && featureChange !== 0) {
+        // Alert if feature changed with no version bump.
+        return `🟥🟥🟥🟥 Feature Change Detected ${newLayer['lds:name']} - version: ${newLayer['lds:version']} features: ${newLayer['lds:feature_count']} (+${featureChange}) 🟥🟥🟥🟥`;
+      }
+
+      if (featureChange >= 0) {
+        // Add Features
+        return `🟦 ${newLayer['lds:name']} - version: ${newLayer['lds:version']} (from: ${existingLayer['lds:version']}) features: ${newLayer['lds:feature_count']} (+${featureChange})`;
```

Comment:
What if `featureChange === 0`? I would assume that would return null.

Reply:
The lds layer might be upgrade version with not new features, maybe just date update on existing feature. In contrast, we shouldn't have a feature change if no version upgrade.

Reply:
That assumption should probably be part of the code.

Reply:
That's the line 182 doing this check first.

Reply:
You mean line 181? That has *two* clauses, so it's entirely possible for `featureChange` to be zero once it gets to this line.